src/transactions/feed/TransactionFeedV2.tsx

```diff
@@ -427,6 +429,17 @@ export default function TransactionFeedV2() {
     [newlyCompletedCrossChainSwaps]
   )
 
+  useEffect(
+    function updatePersistedFeedFirstPage() {
```

Comment:
if i understand this change correctly, the difference lies in that we want to store `firstPageData` instead of the received first page data in redux because `firstPageData` includes the sorted and merged standby transactions? if that is the case, aren't we storing the standby transactions potentially twice?

is it possible instead for the `feedFirstPageSelector` to return the first page that is merged with the standby transactions?

Reply:
@kathaypacific
1. With the implementation from this PR, duplicates will be present but only for pending and failed transactions. Once pending turn to completed - they will get removed from stand by, therefore duplicates will only exist for failed ones but only for the first page of the feed anyways.
2. Yes, it is possible!

Reply:
@kathaypacific Argh, it seems like if I move this to the selector - I won't be able to pass `isLastPage` to the `mergeStandByTransactionsInRange` function. The rest of un-merged stand by transactions are always pushed to the very bottom of the last page but for that we need to somehow figure out if the page is last which is impossible from the selector level.

Reply:
hum, okay. i don't have a better suggestion but am slightly concerned that these small details are causing some complexity / workarounds that we will later forget about. i can see us potentially reintroducing these bugs during cleanup initiatives. perhaps @jeanregisser has a suggestion here, or leaving a small comment with a link to your comment explanation above is sufficient for now

Reply:
@kathaypacific there are a few potential implementation changes that I have in mind but they involve a bit more time to implement as it will most likely require the overall change of managing `paginationData` on the side of Redux rather than on the component level. In that case, we'll have most of the logic on the reducer level and no duplication.